Embroider blog post write up (#967)
ember-learn/ember-blog: content/the-ember-times-issue-180.md

---

## [Embroider: from zero to route splitting in 3.5 weeks 📝](https://dev.to/bendemboski/embroider-from-zero-to-route-splitting-in-3-5-weeks-5abo)

> If you have been part of the Ember community for a while now and have read some of the latest Ember Times you have heard about [Embroider](https://github.com/embroider-build/embroider). This project is currently at version 0.41.0 and usable in Ember projects.  [Ben Demboski @bendemboski](https://github.com/bendemboski) wrote this [blog](https://dev.to/bendemboski/embroider-from-zero-to-route-splitting-in-3-5-weeks-5abo) about how they implemented Embroider in their [`ember-electron`](https://ember-electron.js.org) app.

Reviewer: I think it'd be good to link to last week's issue, as Amy had worked on a Reader's question. What do you think about introducing Ben in a new paragraph?

```suggestion
If you have been part of the Ember community for a while now and have read [The Ember Times' last issue](https://blog.emberjs.com/the-ember-times-issue-179#toc_readers-question-what%E2%80%99s-the-current-status-of-the-embroider-project-%F0%9F%A7%B5), you have probably heard about [Embroider](https://github.com/embroider-build/embroider). This project is currently at version 0.41.0 and usable in Ember projects.

[Ben Demboski @bendemboski](https://github.com/bendemboski) wrote this [blog](https://dev.to/bendemboski/embroider-from-zero-to-route-splitting-in-3-5-weeks-5abo) about how they implemented Embroider in their [`ember-electron`](https://ember-electron.js.org) app.
```

Author: Great suggestion :)

> This [blogpost](https://dev.to/bendemboski/embroider-from-zero-to-route-splitting-in-3-5-weeks-5abo) explains in detail the app and the process they used to make their app use Embroider. All the hard things they ran into like ES6 module compliance and third-party add-ons. Luckily after 3.5 weeks they achieved what they wanted: **tree-shaking** and **code splitting across routes**. So if you want to start taking advantage of what Embroider has to offer, this post is a great place to start.

Reviewer: ```suggestion
Ben's [blog post](https://dev.to/bendemboski/embroider-from-zero-to-route-splitting-in-3-5-weeks-5abo) explains in detail the app and the process they used to make their app use Embroider. All the hard things they ran into, such as ES6 module compliance and third-party add-ons. Luckily, after 3.5 weeks, they achieved what they wanted: **tree-shaking** and **code splitting across routes**. So if you want to start taking advantage of what Embroider has to offer, this post is a great place to start.
```
